cookbook/plans/adventurer.json

```diff
@@ -27,5 +27,5 @@
             }
         ]
     },
-    "deposit": "10000000ulava"
+    "deposit": "10000000000ulava"

```

Comment:
I did it because with mainnet's parameters the deposit was too small. Reverted here: https://github.com/lavanet/lava/commit/082ddb73a47d076bcabf702509e0f36c4932bb10